README.md

```diff
@@ -58,123 +58,130 @@ See [contributing](CONTRIBUTING.md) for more information.
 
 ## Module Robustness & Data Integrity <a name="disclaimer"></a>
-*Calendar entries for this module are pulled from various sources from the net. As such their accuracy cannot be ensured. If you find an incorrect calendar entry (e.g. wrong date, wrong feast type, spelling issue, typos), you are most welcome to contribute to the source code or inform me so that the necessary changes can be made to make this a more robust and reliable module*
 
-*romcal's code logic is developed according to calendar requirements descibed in various church documents sourced from the internet (and even from Wikipedia). If you notice discrepancies between romcal's output and actual liturgical dates, please do contribute your fixes or submit an issue on GitHub.*
+_Calendar entries for this module are pulled from various sources from the net. As such their accuracy cannot be ensured. If you find an incorrect calendar entry (e.g. wrong date, wrong feast type, spelling issue, typos), you are most welcome to contribute to the source code or inform me so that the necessary changes can be made to make this a more robust and reliable module._
+
+_romcal’s code logic is developed according to calendar requirements descibed in various church documents sourced from the internet (and even from Wikipedia). If you notice discrepancies between romcal’s output and actual liturgical dates, please do contribute your fixes or submit an issue on GitHub._
 
 ### Testing romcal
+
 romcal employs `TDD` using `mocha` and `should`.
 Run `npm test` in your console to view test output.
 
-`Travis CI` is used to validate romcal builds to ensure functionality is working as expected.
+Travis CI is used to validate romcal builds to ensure functionality is working as expected.
 
 ## Builds <a name="builds"></a>
 
-Running `npm run build` in the romcal root directory will generate a packaged version of romcal suitable for direct use in a browser.
+Running `npm install` in the romcal root directory will generate a packaged version of romcal suitable for direct use in a browser.
 
 - `dist/romcal.bundle.min.js` A minified and obfruscated bundle of romcal + all its dependencies wrapped in a UMD module shell that is suitable for being included directly in browsers
 
 ## Usage <a name="usage"></a>
 
-> romcal has been re-written using ES6, and use the new import/export syntax to manage node modules. However, thanks to `esm` and `babel`, it can still be included by other non ES6 aware node modules via the normal CommonJs `require` call.
+> :information_source: romcal is written using ES6, and use the new import/export syntax to manage node modules. However, thanks to `esm` and `babel`, it can still be included by other non ES6 aware node modules via the normal CommonJs `require` call.
```

Comment:
here I might prefer the original `has been`. Theses conjugations are exactly the same in french, and I think it's still better than "is written". Furthermore this text has been written by a native english (I guess).

Reply:
I start from the end: @pejulian is Malaysian, so neither is a native English speaker (at least not in the general sence, but I know that English is spoken by most of the Malaysians _at some level_).

I cannot talk about French grammer as I have no knowledge of it.

But on English grammer:
- `has been written` (present perfect tense) is used when something that either:
   - has started in the past and continues in the present (and the re-writing does not continue now);
   - [chiefly British English] started in the past and was done a just before now, but the time when it was done, is not specified (in romcal/Git, the time is always _implicitly_ specified by the commit timestamps);
- `was written` (past tense):
   - when something was done in the past explicitly;
   - it usually implies that it is something (a bit) detached from the present (like a living vs dead person: you talk in present tense of a living person, in past tense of a dead one);
- `is written` (present tense):
   - describes the present state (and as such, it is always true).

I am still pro `is written`.

Reply:
Or `was re-written`, which is also true, but still I think that the `README.md; should contain the description of the _current_ state of romcal, rather than was was done. Of course, if the past things are somewhat important, we might include that too.

Reply:
I know my conjuring, you will not teach me that, especially that the conjuring definitions are the same in French. https://www.quora.com/When-should-you-use-has-been-and-was

But I don't want to waste my time on that.

Reply:
Sorry, I didn’t wanted to upset you nor to teach you anything (though I admit that the approach I’ve chosen is not the best).

I fix if you the way you want it.